### ISSUE TITLE
makeportable.bat: still has `ERROR: CoreAudioToolbox.dll: The specified module could not be found.`

### ISSUE THREAD

**Issue by vitaly-zdanevich** [nu774/qaac#59]
In the same directory I has `qaac64.exe` and resulting `QTfiles64` with that files inside:
```
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         3/9/2019   5:18 AM          80696 ASL.dll
-a----         3/9/2019   5:18 AM        9039672 CoreAudioToolbox.dll
-a----         3/9/2019   5:18 AM        2229560 CoreFoundation.dll
-a----         3/9/2019   5:18 AM       27159352 icudt62.dll
-a----         3/9/2019   5:18 AM         104760 libdispatch.dll
-a----         3/9/2019   5:18 AM        2457912 libicuin.dll
-a----         3/9/2019   5:18 AM        1579320 libicuuc.dll
-a----        6/11/2011   1:15 AM         608080 msvcp100.dll
-a----        6/11/2011   1:15 AM         829264 msvcr100.dll
-a----         3/9/2019   5:18 AM         237368 objc.dll
-a----       11/20/2018  10:30 PM        2006528 qaac64.exe
```

**Comment by nu774** [comment]
Can't you still run qaac?
If you could, please close the issue.

**Comment by vitaly-zdanevich** [comment]
I installed iTunes :( and yes now I can use `qaac`

**Comment by nu774** [comment]
It seems that recent AppleApplicationSupport DLLs are build with Visual Studio 2015 and are dependent on UniversalCRT, VCRUNTIME140.dll and MSVCP140.dll.
UniversalCRT is part of OS since Windows 10. If you are still using Windows 7 ～ 8.1 and your OS is updated by Windows Update, you are likely to already have them. See here: https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows
As for  VCRUNTIME140.dll and MSVCP140.dll, they are Visual Studio 2015 specific runtime, and you can download from https://www.microsoft.com/en-us/download/details.aspx?id=48145.

**Comment by nu774** [comment]
Now updated makeportable for iTunes 12.9.4+.
Please try with it.

**Comment by istomyang** [comment]
> It seems that recent AppleApplicationSupport DLLs are built with Visual Studio 2015 and are dependent on UniversalCRT, VCRUNTIME140.dll and MSVCP140.dll.
> UniversalCRT is part of OS since Windows 10. If you are still using Windows 7 ～ 8.1, See here: https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows
> As for VCRUNTIME140.dll and MSVCP140.dll, they are Visual Studio 2015 specific runtime, and you can download from https://www.microsoft.com/en-us/download/details.aspx?id=48145.

It works! Thank you.

**Comment by odinian** [comment]
sorry to hijack an old thread, but I installed iTunes from the MS store a long time ago and it works. I installed qaac and I get the "ERROR: CoreAudioToolbox.dll: The specified module could not be found." error (qaac64 too). I just download the latest iTunes installer iTunes64Setup.exe and the latest makeportable, and it created the QTfiles64 folder. I put the QTfiles64 in the same folder as the Qaac64.exe executable, and I still get the "ERROR: CoreAudioToolbox.dll: The specified module could not be found." error when I try qaac64 --check
Any ideas?

**Comment by ppw0** [comment]
Yeah, this isn't working for me at all, no matter how many versions of how many C/C++ runtimes I cram into the same folder.

**Comment by gevvan** [comment]
Any news?

```
c:\qaac>qaac64 --check
ERROR: CoreAudioToolbox.dll: The specified module could not be found.
```

Still!

**Comment by istomyang** [comment]
@gevvan Use supporting files in https://github.com/AnimMouse/QTFiles
